cli/remove/remove.go

```diff
@@ -0,0 +1,71 @@
+package remove
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/apex/log"
+	"github.com/tarantool/tt/cli/util"
+)
+
+// Remove removes binary/directory and symlinks.
+func Remove(program string, dst string) error {
+	if dst == "" {
+		dst, _ = os.Getwd()
+		dst = filepath.Join(dst, "bin")
+	}
+	path := filepath.Join(dst, program)
+
+	var linkPath string
+	var err error
+	if strings.HasPrefix(program, "tt") {
+		linkPath, err = util.JoinAbspath(dst, "/tt")
+		if err != nil {
+			return err
+		}
+	} else if strings.HasPrefix(program, "tarantool") {
+		linkPath, _ = util.JoinAbspath(dst, "/tarantool")
+	} else {
+		return fmt.Errorf("Unknown program: %s", program)
+	}
+
+	if _, err := os.Stat(path); os.IsNotExist(err) {
+		return fmt.Errorf("There is no %s installed.", program)
+	}
+	log.Infof("%s found, removing...", program)
+
+	// Get path where symlink point.
+	resolvedPath, err := util.ResolveSymlink(linkPath)
+	if err != nil {
+		return fmt.Errorf("There was some problem with resolving symlink path: %s", err)
+	}
+	// Remove symlink if it points to program.
+	if !strings.Contains(resolvedPath, program) {
+		err = os.RemoveAll(path)
+	} else {
+		err = os.RemoveAll(linkPath)
```

Comment:
Why do we use RemoveAll for symbolic link?

Reply:
changed